ivy/functional/frontends/scipy/spatial/distance.py

```diff
@@ -30,26 +30,14 @@ def _validate_weights(w, dtype="float64"):
 
 # minkowski
 @to_ivy_arrays_and_back
-def minkowski(u, v, p=2, /, *, w=None):
-    u = _validate_vector(u)
-    v = _validate_vector(v)
+def minkowski(u: List[float], /, *, v: List[float], p: float = 2, w: Optional[float] = None) -> float:
+    u,v = _validate_vector(u),_validate_vector(v)
     if p <= 0:
         raise ValueError("p must be greater than 0")
     u_v = u - v
-    if w is not None:
-        w = _validate_weights(w)
-        if p == 1:
-            root_w = w
-        elif p == 2:
-            # better precision and speed
-            root_w = ivy.sqrt(w)
-        elif p == ivy.inf:
-            root_w = w != 0
-        else:
-            root_w = ivy.pow(w, 1 / p)
-        u_v = ivy.multiply(root_w, u_v)
-    dist = sc_frontend.linalg.norm(u_v, ord=p)
-    return dist
+    root_w = {1: w, 2: ivy.sqrt(w), ivy.inf: w != 0}.get(p, w ** (1/p) if w else None)
+    u_v = [ivy.multiply(root_w, elem) if root_w is not None else elem for elem in u_v]
+    return ivy_np.linalg.norm(u_v, ord=p)
 
```

Comment:
To functionally shorten this implementation according to the guidelines of the task, `minkowski` should be added to our api and then used in the frontend. To do this you would need to open a new PR to implement `ivy.minkowski` first. Otherwise feel free to just close this and take on another task.